docs/ad/index.md

```diff
@@ -40,36 +45,54 @@ A feature is the field in your index that you want to find anomalies for. A dete
 
 Anomaly grade is a number between 0 and 1 that indicates the level of severity of how anomalous a data point is. An anomaly grade of 0 represents “not an anomaly,” and a non-zero value represents the relative severity of the anomaly. The confidence score is an estimate of the probability that the reported anomaly grade matches the expected anomaly grade. Confidence increases as the model observes more data and learns the data behavior and trends. Note that confidence is distinct from model accuracy.
 
-The **Features** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
+The **Feature breakdown** graph plots the features based on the aggregation method. On the top-right corner, you can vary the date-time range of the detector.
+
+### Step 4: Set up alerts
 
-To see all the anomalies that are detected, choose the **Anomalies** tab.
+To create a monitor to send you notifications when any anomalies are detected, choose **Set up alerts**.
+You're redirected to the **Alerting**, **Add monitor** page.
 
-### Step 4: Adjust the model
+For steps to create a monitor and set notifications based on your anomaly detector, see [Monitor](../alerting/monitors/).
 
-You can enable or disable features or fine tune the time interval to minimize any false positives.
+If you stop or delete a detector, make sure to delete any monitors associated with the detector, as this is not done automatically.
 
-1. Choose **Adjust Model**.
-2. In **Global Settings**, specify the time interval in minutes.
+### Step 5: Adjust the model
 
-### Step 5: Create a monitor
+To see all the configuration settings, choose the **Detector configuration** tab.
 
-To create a monitor to send you notifications when any anomalies are detected, choose **Actions** and select **Create Monitor**. For steps to create a monitor and set notifications based on your anomaly detector, see [Monitor](../alerting/monitors/).
+1. To make any changes to the detector configuration, in the **Detector configuration** section, choose **Edit**
+1. To enable or disable features or fine tune the time interval to minimize any false positives, in the **Features** section, choose **Edit**.
 
 ### Step 6: Manage your detectors
 
-You can change or delete your detectors.
+Go to the **Detectors** page to change or delete your detectors.
 
-1. To make changes to your detector, choose **Actions**, and then choose **Edit detector**. After completing your changes, choose **Update**.
-2. To delete your detector, choose **Actions**, and then choose **Delete detector**. Choose **Yes** to confirm.
+1. To make changes to your detector, choose **Actions**, and then choose **Edit detector**. You need to *stop* the detector before changing the configuration settings. After completing your changes, choose **Save change**.
+2. To delete your detector, choose **Actions**, and then choose **Delete detector**. Choose **Delete** to confirm.
```

Comment:
Same here, need to delete detector on detector detail page

Reply:
Done.